code/modules/vehicles/droppod.dm

```diff
@@ -0,0 +1,223 @@
+/obj/vehicle/droppod
+	name = "drop pod"
+	desc = "A big metal pod, what could be inside?"
+	icon = 'icons/obj/bike.dmi'
+	icon_state = "bike_off"
+	dir = SOUTH
+
+	load_item_visible = 0
+	health = 500 // pretty strong because it can't move or be shot out of
+	maxhealth = 500
+
+	fire_dam_coeff = 0.6
+	brute_dam_coeff = 0.5
+	var/list/validfirelocations = list(7)
+
+	var/used = 0
+
+	var/mob/humanload 
+	var/mob/passenger
+
+	var/list/protectedareas = list(/area/hallway/secondary/entry/dock, /area/crew_quarters/sleep/cryo, /area/crew_quarters/sleep/bedrooms)
+
+/obj/vehicle/droppod/Initialize()
+	. = ..()
+
+/obj/vehicle/droppod/emag_act()
+	return
+
+/obj/vehicle/droppod/Move()
+	return
+
+/obj/vehicle/droppod/load(var/mob/C) // this won't call the parent proc due to the differences and the fact it doesn't use load. Also only mobs can be loaded.
+	if(!ismob(C))
+		return
```

Comment:
`return 0`

Otherwise this is `return null`, but this function should return boolean.